### PR TITLE
test on 16 and 17 separately

### PR DESCRIPTION
**Pull request by minerva1993**
- Change paths to CMSSW_BASE / relative paths
- update cross section yml
- check default settings for 2017
- plotIt is going under HEPToolsFCNC (can use git pull in the directory)
- PostFitShapes -> PostFitShapesFromWorkspace 
- decorrelate PU, remove prefire (due to typo...will be restored soon)
- some typos

### CODE REVIEW

File: prepareShapesAndCards.py [hunk 1]
```diff
@@ -31,25 +31,26 @@ def setNegativeBinsToZero(h, process):
...
+parser.add_argument('--sysToAvoid', action='store', dest='sysToAvoid', nargs='+', default=['prefire'], help='Set it to exclude some of the systematics. Name should as in rootfile without the up/dowm postfix')
 # Example to call it: python prepareShapesAndCards.py --sysToAvoid pu hf
 parser.add_argument('--sysForSMtt', action='store', dest='sysForSMtt', nargs='+', default=['scale', 'TuneCP5', 'ps', 'pdf','hdamp'], help='Systematics affecting only SM tt.')
-parser.add_argument('--correlatedSys', action='store', dest='correlatedSys', nargs='+', default=['pu', 'scale', 'TuneCP5', 'ps', 'pdf','hdamp'], help='Systematics that are correlated accross years. NB: cross section unc are added by hand at the end of this script, go there to change correlation for them.')
+parser.add_argument('--correlatedSys', action='store', dest='correlatedSys', nargs='+', default=['scale', 'TuneCP5', 'ps', 'pdf','hdamp'], help='Systematics that are correlated accross years. NB: cross section unc are added by hand at the end of this script, go there to change correlation for them.')
```
Comment (BrieucF) [inline prepareShapesAndCards.py:53]: Are you sure about that? The way we derive it is by varying the minBias cross section, did you check that the were used from different prediction/measurements in 2016, 2017 and 2018?

File: xsec_2018_190702.yml [hunk 2]
```diff
@@ -72,13 +70,13 @@
   legend: 'TTHut'
   order: 4
 
-'hist_SingleMuonRun2017.root':
+'hist_SingleMuonRun2018.root':
```
Comment (BrieucF) [inline xsec_2018_190702.yml]: I agree it is better for consistency, though these lines are not used for data, could simply discard them if it makes nothing crash

File: xsec_2018_190702.yml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 'hist_STTH1L3BHct.root':
   type: signal
   pretty-name: 'STTH1L3BHct'
-  cross-section: 0.04
-  generated-events: 3016710
+  cross-section: 0.004
```
Comment (BrieucF) [inline xsec_2018_190702.yml]: Why is that?

File: prepareShapesAndCards.py (hunk 1 above)
Comment (minerva1993) [reply prepareShapesAndCards.py:53]: We use same procedure but the templates for MC true PV are different. In four top analysis as well as a report from Rochester in a top cross section meeting, PU systematic was decorrelated

File: xsec_2018_190702.yml (hunk 2 above)
Comment (minerva1993) [reply xsec_2018_190702.yml]: This should not affect the code. It can be safely removed

File: xsec_2018_190702.yml (hunk 3 above)
Comment (minerva1993) [reply xsec_2018_190702.yml]: It's a typo indeed

**Comment by BrieucF** [comment]
Thanks!

**Pull request merged**